enhancements/gateway/20220130-distinguishing-central-agent.md

```diff
@@ -0,0 +1,88 @@
+# Distinguishing the central Opni cluster agent
+
+## Summary:
+This proposal describes a mechanism by which the agent running in the central opni cluster can be distinguished from other clusters.
+
+## Use case:
+This is a prerequisite for several other features. For example, identifying the central opni cluster would be required for implementing a global metrics tenant.
+
+## Benefits:
+The ability to distinguish the central cluster from other clusters provides benefits such as:
+- It allows establishing an increased level of trust with a specific agent, allowing it to be authorized for tasks that cannot be entrusted to other agents.
+- It allows the UI, CLI, or other API consumers to distinguish the central cluster from other clusters, which is helpful for users.
```

Comment:
This proposal should include all UI enhancements that can come out of this feature. The one that is most obvious is that the UI distinguishes the Opni cluster's agent in the UI. Anything else?
@codyrancher Can we add an estimate for this to the level of effort section ?

Reply:
The only planned UI change will be adding some sort of distinguishing mark or label to the central cluster.

Reply:
We discussed with Cody. UI will include a bubble to distinguish local agent